src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/asides/ts.ui.AsideSpirit.js

```diff
@@ -372,33 +364,28 @@ ts.ui.AsideSpirit = (function using(chained, confirmed, Client, LayoutModel, not
 			 * TODO: Validate that we are not opening inside .ts-main
 			 * @param {boolean} animated (not supported just yet)
 			 */
-			_open: function(animated) {
-				if (this.super._open(animated)) {
-					this.super._open(animated);
-					this._delayedAngularInitialization();
-					this._trapattention();
-					this._willopen();
-					this._slideopen(true).then(
-						function done() {
-							this._ontransitionend();
-						}.bind(this)
-					);
-				}
+			$onopen: function(animated) {
+				this._delayedAngularInitialization();
```

Comment:
I wish we didn't have this. It's not always needed, there was some cornercase in Angular with complex, nested templates, but I'm afraid to remove it now.